app/models/server/models/LivechatSessions.js

```diff
@@ -0,0 +1,154 @@
+import { Base } from './_Base';
+
+/**
+ * Livechat session model
+ */
+
+export class LivechatSessions extends Base {
+	constructor() {
+		super('livechat_sessions');
+	}
```

Comment:
Add a unique index here:

`this.tryEnsureIndex({ token: 1 }, { unique: true });`